internal/cmd/root.go

```diff
@@ -127,16 +127,7 @@ func (c *rootCommand) execute() {
 }
 
 // Execute adds all child commands to the root command sets flags appropriately.
-// This is called by main.main(). It only needs to happen once to the rootCmd.
-func Execute() {
-	gs := state.NewGlobalState(context.Background())
-	newRootCommand(gs).execute()
-}
-
-// ExecuteWithGlobalState runs the root command with an existing GlobalState.
-// This is needed by integration tests, and we don't want to modify the
-// Execute() signature to avoid breaking k6 extensions.
-func ExecuteWithGlobalState(gs *state.GlobalState) {
+func Execute(gs *state.GlobalState) {
```

Comment:
Can we isolate this change in a separated pull request, please? It doesn't sound really a requirement to have it here. Or am I missing something?

Reply:
This is done